s2d-oms-mgmt-solution/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "omsLogAnalyticsWorkspaceName": {
+            "value": "changeme"
+        },
+        "omsLogAnalyticsWorkspaceLocation": {
+            "value": "East US"
+        },
+        "omsLogAnalyticsWorkspaceApiVersion":{
```

Comment:
We generally recommend having api versions as raw strings in the template unless we expect users to change this frequently; is this the case?